### PR TITLE
Add friend requests email sample

### PR DESCRIPTION
**Pull request by fstanis**

### CODE REVIEW

File: src/amphtml-email/20_Samples/Friend_Requests_Email.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
+<!---{
+ "preview": "amp4email",
+ "skipValidation": "true",
+ "draft": "true"
+}--->
+<!--
+## Introduction
+
+This sample showcases how to use AMPHTML email to create an email that lets
+you approve or delete of friend requests on a social network.
```
Comment (kul3r4) [inline src/amphtml-email/20_Samples/Friend_Requests_Email.html]: delete of friend -> delete friend

File: src/amphtml-email/20_Samples/Friend_Requests_Email.html [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!--
+    The AMPHTML Email boilerplate.
+  -->
+  <style amp4email-boilerplate>body{visibility:hidden}</style>
+  <title>AMPHTML Email - Friend Requests</title>
+  <!-- ## Setup -->
+  <!--
+    Additionally used AMP components must be imported in the header. We use
+    `amp-form` to communicate the user's choice to the server.
```
Comment (kul3r4) [inline src/amphtml-email/20_Samples/Friend_Requests_Email.html]: to communicate -> to submit

File: src/amphtml-email/20_Samples/Friend_Requests_Email.html [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+      padding-right: 2em;
+      font-size: 2em;
+    }
+  </style>
+</head>
+<!-- ## Body -->
+<!-- -->
+<body>
+  <!--
+    The list of friend requests is fetched remotely from the server:
```
Comment (kul3r4) [inline src/amphtml-email/20_Samples/Friend_Requests_Email.html]: remove ":"

File: src/amphtml-email/20_Samples/Friend_Requests_Email.html [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+</html>
+
+<!--
+  ## Server-side considerations
+
+  Because there is no authentication for outgoing XHR calls from AMP email
+  messages, all authentication should be done using access tokens.
+
+  In the sample above, `REPLACE_WITH_YOUR_TOKEN` represents an access token.
+  This token has to be provided both in the `src` field inside `<amp-list>` and
```
Comment (kul3r4) [inline src/amphtml-email/20_Samples/Friend_Requests_Email.html]: this token has to -> this token must be

**Comment by kul3r4** [comment]
@fstanis I can merge this PR on master, right?

**Comment by fstanis** [comment]
I think it's fine, especially since it has `draft: true`.

@juliantoledo WDYT?

**Comment by juliantoledo** [comment]
SGTM.

**Pull request merged**